reactDashboard/src/components/HeatMap/Heatmap.css

```diff
@@ -1,35 +1,57 @@
 .heatmap {
   overflow-y: auto;
+  overflow-x: auto;
```

Comment:
We don't want a horizontal overflow here. The heatmap table should work without horizontal scrolling all the way down to about 360px wide screen size.

Reply:
✅